### PR TITLE
umurmur: Update to lastest version. Replace PolarSSL with mbed TLS

### PR DESCRIPTION
**Pull request by thess**
Maintainer: @fatbob313 
Compile tested: AR71xx
Run tested: Not a user of this package

Description:

* Update to latest source for mbed TLS support.
* Supersedes #3789 

Signed-off-by: Ted Hess <thess@kitschensync.net>

### CODE REVIEW

File: net/umurmur/Makefile [hunk 1]
```diff
@@ -7,14 +7,14 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=umurmur
-PKG_VERSION:=0.2.16
+PKG_VERSION:=0.2.16-20161126
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
```
Comment (diizzyy) [inline net/umurmur/Makefile]: xz Please :-)

File: net/umurmur/Makefile [hunk 2]
```diff
@@ -7,14 +7,14 @@
...
 PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
 PKG_SOURCE_URL:=git://github.com/umurmur/umurmur.git
 PKG_SOURCE_PROTO:=git
-PKG_SOURCE_VERSION:=0.2.16
+PKG_SOURCE_VERSION:=fac630135664fda26338d403041b6b6e558a2d6b
```
Comment (diizzyy) [inline net/umurmur/Makefile]: Add PKG_MIRROR_MD5SUM:= ?

File: net/umurmur/Makefile [hunk 3]
```diff
@@ -7,14 +7,14 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=umurmur
-PKG_VERSION:=0.2.16
+PKG_VERSION:=0.2.16-20161126
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
 PKG_SOURCE_URL:=git://github.com/umurmur/umurmur.git
```
Comment (diizzyy) [inline net/umurmur/Makefile]: Use HTTPS

**Comment by thess** [comment]
All done - waiting for maintainer @fatbob313 OK

**Comment by hnyman** [comment]
I compile tested this for ipq806x

fatbob313 has not been active since 2015...

**Comment by thess** [comment]
I'll merge this cleanup but I don't want to be maintainer.

**Pull request merged**